dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -276,6 +281,22 @@ def change_config(python, backend, cheatsheet, asciiart):
         console.print(f'[blue]Backend cached_value {backend}')
 
 
+@option_verbose
+@main.command(name="static-check")
+@click.option('-t', '--type', type=click.Choice(PRE_COMMIT_LIST), multiple=True)
+def static_check(verbose: bool, type: Tuple[str]):
```

Comment:
I am tempted to add few more options here, some that make it a bit easier to use in common cases:

* `--all-files` ->  should also add `--all-files` for the pre-commit command
* `--last-commit` -> this one I found I use quite often and it should add those params: `--from-ref` `HEAD^` `--to-ref` `HEAD`